### DISCUSSION TITLE
Make beatmap titles open the web version

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17531] [Ideas]
![image](https://user-images.githubusercontent.com/40531871/160660058-3a18a2b4-2202-4894-8f65-655fb2ff7a81.png)

Can you make the title clickable like on the mapper's name to check the web version?
Currently we have to click on the mapper's username & search on his profile

**Comment by Theighlin** [comment]
You can right click the map on the carousel and click details

**Reply by AuracleTech** [reply]
Strangely enough I noticed some maps aren't linked to the web, so I'm trying to figure out why
Lots of my maps don't have details ( as if they were updated or something ) I need to verify

**Reply by frenzibyte** [reply]
Can you please link said maps. If they don't have the "Details" tab then the online ID is somehow not attached to the beatmap.

**Reply by AuracleTech** [reply]
Just checked lots of my maps and they're ok
It might be an impression due to the missing quick link, pretty sure in an old version of lazer the short link was there like on the web version, or maybe I just misremember
[This one](https://osu.ppy.sh/beatmapsets/1322944#osu/2988681) is definitively not linked, but I imported from stable a month ago so that's probably it.

Still could be cool if we could click the name, we can in multiplayer, which is much better than right click details to be honest, but both could be nice

**Comment by peppy** [comment]
At most the title would have a way to open the details overlay, not a direct web link.

**Reply by AuracleTech** [reply]
If you're talking about the in-game web then yes that's what I meant 😁

**Reply by peppy** [reply]
Web = browser. In-game != web so please don't call it that to avoid confusion.
